tests/console/yast2_nfs4_server.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2015-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add new yast2_nfs_server test
+#    This tests "yast2 nfs-server" by creating an NFS share,
+#    writing a file to it and validating that the file is accessible
+#    after mounting.
+#    It can also be used as a server in an "/ on NFS" test scenario.
+#    In this case, NFS4SERVER has to be 1, the server is accessible as
+#    10.0.2.1 and it provides a mutex "nfs4_ready".
+# Maintainer: Fabian Vogt <fvogt@suse.com>
+
+use base "y2_module_consoletest";
+
+use strict;
+use warnings;
+use utils qw(clear_console zypper_call systemctl);
+use version_utils;
+use testapi;
+use lockapi;
+use mmapi;
+use mm_network;
+use nfs_common;
+
+sub run {
+    my ($self) = @_;
+    my $rw     = '/srv/nfs';
+    my $ro     = '/srv/nfs/ro';
+    select_console 'root-console';
+
+    if (get_var('NFS4SERVER')) {
```

Comment:
if I don't miss something `yast2_nfs_server` is loaded only when NFS4SERVER is defined , so there is no point to check if it is defined again second time

Reply:
The point is, the `yast2_nfs_server` which I started from was used also as a standalone test (not multimachine) and that's why I kept such option also for `yast2_nfs4_server`. Perhaps it's new test perhaps we want to use it as multimachine (with the client) this option is not used yet but it can be used in the future.

Reply:
sorry I did a typo in my first message which might confuse you : 

> if I don't miss something yast2_nfs**4**_server is loaded only when NFS4SERVER is defined , so there is no point to check if it is defined again second time

my point that to get into this file currently only way is via `sle/main.pm` line 1014-1015 . So we **know** that NFS4SERVER=1 . And we can't support or discuss something that might or might not implemented one day

Reply:
Yes, you are absolutely right, I will remove those statements then.